packages/jest-snapshot/src/__tests__/printSnapshot.test.ts

```diff
@@ -34,6 +36,11 @@ import {
 } from '../printSnapshot';
 import {serialize} from '../utils';
 
+type SyncExpectationResult = {
```

Comment:
Or would it be better to expose it from `expect`?

Reply:
yes